tests/unit/sagemaker/image_uris/test_autogluon.py

```diff
@@ -60,6 +60,8 @@
     "0.6.2",
     "0.7.0",
     "0.7",
+    "0.8",
```

Comment:
Can you please run `tox -e black-check` 

ref: https://github.com/aws/sagemaker-python-sdk/blob/master/.githooks/pre-push

Reply:
Addressed